plugins/ssrmatching/ssrmatching.ml

```diff
@@ -689,15 +695,14 @@ let mk_tpattern_matcher ?(all_instances=false)
        | _ -> false)
     | _ -> unif_EQ env sigma u.up_f in
 let p2t p = mkApp(p.up_f,p.up_a) in 
-let source env = match upats_origin, upats with
+let source env sigma = match upats_origin, upats with
   | None, [p] -> 
       (if fixed_upat ise p then str"term " else str"partial term ") ++
-      pr_constr_pat env ise (p2t p) ++ spc()
+      pr_constr_pat env sigma (p2t p) ++ spc()
```

Comment:
In the two lines above `p` is used in both `sigma` and `ise` while before it was always used in `ise`.
Please revert this, there is no need to abstract source on sigma.
BTW, this is code I've asked maxime to fix already in his branch...

Reply:
Done, indeed thanks, I was aware of what happened but I didn't remember why I did abstract source so I thought I would get additional review.